public/app/features/dashboard-scene/edit-pane/DashboardEditPaneSplitter.tsx

```diff
@@ -68,7 +68,7 @@ export function DashboardEditPaneSplitter({ dashboard, isEditing, body, controls
   }
 
   const onBodyRef = (ref: HTMLDivElement) => {
-    dashboard.onSetScrollRef(ref);
+    dashboard.onSetScrollRef(new DivScrollElement(ref));
```

Comment:
Setting the scroll element here helps to use the defined `scrollTo` method which also has a retry system, instead of the native scrollTo which will just fail with no retry, thus having the issue.